### ISSUE TITLE
Pre-work Checklist: Developer: Matt Pereira

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3602]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by MattPereira** [comment]
1. **Progress**: Completed most of the checklist items but still need to attend another weekly meeting and dive into Figma
2. **Blockers**: No difficulties or errors encountered
3. **Availability**: 6 hours
4. **ETA**: End of day 10/20/22
5. **Pictures**: No visual changes made to site yet

**Comment by HackforLABot** [comment]
## Activity Log: MattPereira
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- MattPereira opened: https://github.com/hackforla/website/issues/3602 at 10/10/2022, 07:41 PM PDT
- MattPereira assigned: https://github.com/hackforla/website/issues/2845#issuecomment-1048146239 at 10/11/2022, 08:01 PM PDT
- MattPereira assigned: https://github.com/hackforla/website/issues/3602 at 10/11/2022, 08:01 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/2845#issuecomment-1275523189 at 10/11/2022, 08:07 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/2845#issuecomment-1275556981 at 10/11/2022, 09:05 PM PDT
- MattPereira opened: https://github.com/hackforla/website/pull/3615 at 10/11/2022, 09:24 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3616#issuecomment-1277930112 at 10/13/2022, 10:12 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3616#pullrequestreview-1141232404 at 10/13/2022, 10:35 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3617#issuecomment-1278246221 at 10/13/2022, 03:30 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3617#pullrequestreview-1141597861 at 10/13/2022, 03:51 PM PDT
- MattPereira PR was merged: https://github.com/hackforla/website/pull/3615#event-7586643290 at 10/13/2022, 08:26 PM PDT
- MattPereira issue closed by PR 3615: https://github.com/hackforla/website/issues/2845#event-7586643404 at 10/13/2022, 08:26 PM PDT
- MattPereira assigned: https://github.com/hackforla/website/issues/3199#issuecomment-1145809380 at 10/14/2022, 10:00 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/3199#issuecomment-1279254932 at 10/14/2022, 10:04 AM PDT
- MattPereira opened: https://github.com/hackforla/website/pull/3621 at 10/14/2022, 11:28 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/3602#issuecomment-1279349271 at 10/14/2022, 11:50 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3620#issuecomment-1279784727 at 10/15/2022, 10:07 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3620#pullrequestreview-1143209654 at 10/15/2022, 10:15 AM PDT
- MattPereira PR was merged: https://github.com/hackforla/website/pull/3621#event-7595484535 at 10/15/2022, 01:35 PM PDT
- MattPereira issue closed by PR 3621: https://github.com/hackforla/website/issues/3199#event-7595484562 at 10/15/2022, 01:35 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3623#pullrequestreview-1143228082 at 10/15/2022, 02:09 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3623#issuecomment-1279833146 at 10/15/2022, 02:12 PM PDT
- MattPereira assigned: https://github.com/hackforla/website/issues/3493#issuecomment-1253988018 at 10/17/2022, 12:56 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/3493#issuecomment-1281444514 at 10/17/2022, 01:20 PM PDT
- MattPereira opened: https://github.com/hackforla/website/pull/3633 at 10/17/2022, 04:02 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3629#issuecomment-1281616663 at 10/17/2022, 04:15 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3629#pullrequestreview-1144917021 at 10/17/2022, 04:29 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3633#issuecomment-1282812121 at 10/18/2022, 11:11 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3642#issuecomment-1284435580 at 10/19/2022, 11:49 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3642#pullrequestreview-1148129534 at 10/19/2022, 12:00 PM PDT
- MattPereira PR was merged: https://github.com/hackforla/website/pull/3633#event-7626235179 at 10/19/2022, 05:51 PM PDT
- MattPereira issue closed by PR 3633: https://github.com/hackforla/website/issues/3493#event-7626235255 at 10/19/2022, 05:51 PM PDT
- MattPereira assigned: https://github.com/hackforla/website/issues/2292#issuecomment-923623422 at 10/20/2022, 11:22 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/2292#issuecomment-1285967312 at 10/20/2022, 11:24 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3643#issuecomment-1285972086 at 10/20/2022, 11:28 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3643#pullrequestreview-1149819056 at 10/20/2022, 12:03 PM PDT
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3602#event-7635047472 at 10/20/2022, 02:14 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3645#issuecomment-1286247244 at 10/20/2022, 03:53 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3645#pullrequestreview-1150104581 at 10/20/2022, 03:57 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/2292#issuecomment-1287477116 at 10/21/2022, 02:59 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3648#issuecomment-1287891883 at 10/22/2022, 12:31 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3648#pullrequestreview-1152118791 at 10/22/2022, 12:43 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3649#issuecomment-1288236475 at 10/23/2022, 04:59 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3649#issuecomment-1288354696 at 10/23/2022, 08:11 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3649#pullrequestreview-1153486474 at 10/24/2022, 10:02 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3650#issuecomment-1289425225 at 10/24/2022, 11:29 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3650#pullrequestreview-1153594592 at 10/24/2022, 11:32 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/2292#issuecomment-1289558836 at 10/24/2022, 01:16 PM PDT
- MattPereira opened: https://github.com/hackforla/website/pull/3653 at 10/24/2022, 03:51 PM PDT
- MattPereira closed: https://github.com/hackforla/website/pull/3653#event-7667493274 at 10/25/2022, 02:56 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3656#issuecomment-1291194981 at 10/25/2022, 03:08 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3659#issuecomment-1291263502 at 10/25/2022, 05:00 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3659#pullrequestreview-1155694201 at 10/25/2022, 05:09 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3660#issuecomment-1291269729 at 10/25/2022, 05:11 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3664#issuecomment-1294098005 at 10/27/2022, 02:40 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3652#issuecomment-1294223661 at 10/27/2022, 04:17 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3652#pullrequestreview-1159153613 at 10/27/2022, 04:34 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3664#pullrequestreview-1159169420 at 10/27/2022, 05:07 PM PDT
- MattPereira opened: https://github.com/hackforla/website/pull/3672 at 10/28/2022, 12:26 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3662#issuecomment-1295394616 at 10/28/2022, 12:44 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3662#pullrequestreview-1160629616 at 10/28/2022, 12:53 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3673#issuecomment-1295930052 at 10/29/2022, 11:33 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3673#pullrequestreview-1161028138 at 10/29/2022, 11:45 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3672#issuecomment-1297463689 at 10/31/2022, 10:58 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3677#issuecomment-1297478751 at 10/31/2022, 11:11 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3677#pullrequestreview-1162334755 at 10/31/2022, 11:16 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3649#pullrequestreview-1162406561 at 10/31/2022, 12:10 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3692#issuecomment-1302405492 at 11/03/2022, 09:58 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3692#pullrequestreview-1167658563 at 11/03/2022, 02:01 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3672#issuecomment-1304598171 at 11/05/2022, 10:40 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3695#issuecomment-1304641075 at 11/05/2022, 02:02 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/3696#issuecomment-1304641187 at 11/05/2022, 02:03 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/3695#pullrequestreview-1169647219 at 11/06/2022, 12:21 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3696#pullrequestreview-1169761594 at 11/06/2022, 07:48 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3672#issuecomment-1306094209 at 11/07/2022, 11:36 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3672#issuecomment-1307500170 at 11/08/2022, 08:35 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3699#issuecomment-1307975824 at 11/08/2022, 03:30 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3699#pullrequestreview-1172990889 at 11/08/2022, 03:32 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3702#issuecomment-1308098746 at 11/08/2022, 06:06 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3702#pullrequestreview-1174689018 at 11/09/2022, 12:14 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3705#issuecomment-1310766741 at 11/10/2022, 11:03 AM PST
- MattPereira PR was merged: https://github.com/hackforla/website/pull/3672#event-7799696505 at 11/13/2022, 05:48 PM PST
- MattPereira issue closed by PR 3672: https://github.com/hackforla/website/issues/2292#event-7799696550 at 11/13/2022, 05:48 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/2292#issuecomment-1312941872 at 11/13/2022, 06:06 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3334#issuecomment-1176466870 at 11/13/2022, 06:20 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1312953666 at 11/13/2022, 06:24 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3717#issuecomment-1314440859 at 11/14/2022, 01:39 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3717#pullrequestreview-1180252349 at 11/14/2022, 10:13 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3719#issuecomment-1315784715 at 11/15/2022, 11:43 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3723 at 11/15/2022, 08:44 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3723#issuecomment-1316378690 at 11/15/2022, 09:26 PM PST
- MattPereira closed as not planned: https://github.com/hackforla/website/issues/3723#event-7820092920 at 11/15/2022, 09:26 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3719#pullrequestreview-1183172161 at 11/16/2022, 11:03 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1186760073 at 11/18/2022, 12:46 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1186862833 at 11/18/2022, 02:02 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3729#pullrequestreview-1186888639 at 11/18/2022, 02:25 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3729#event-7847907318 at 11/18/2022, 02:26 PM PST
- MattPereira closed by PR 3729: https://github.com/hackforla/website/issues/3211#event-7847907387 at 11/18/2022, 02:26 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1187018138 at 11/18/2022, 08:31 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3726#pullrequestreview-1187136969 at 11/19/2022, 03:05 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3726#pullrequestreview-1187138018 at 11/19/2022, 03:14 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3733#issuecomment-1322783295 at 11/21/2022, 03:15 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3733#event-7861975132 at 11/21/2022, 03:17 PM PST
- MattPereira closed by PR 3733: https://github.com/hackforla/website/issues/3212#event-7861975227 at 11/21/2022, 03:18 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3742 at 11/22/2022, 07:41 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3742#event-7873348662 at 11/22/2022, 07:49 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3742#issuecomment-1324519314 at 11/22/2022, 08:01 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3726#pullrequestreview-1193781092 at 11/24/2022, 07:10 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3726#pullrequestreview-1193784139 at 11/24/2022, 07:18 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1194768854 at 11/25/2022, 01:13 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1194775046 at 11/25/2022, 01:26 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3745#pullrequestreview-1194786747 at 11/25/2022, 02:18 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3745#event-7895318895 at 11/25/2022, 02:19 PM PST
- MattPereira closed by PR 3745: https://github.com/hackforla/website/issues/3213#event-7895318937 at 11/25/2022, 02:19 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3726#event-7896601931 at 11/26/2022, 10:43 AM PST
- MattPereira closed by PR 3726: https://github.com/hackforla/website/issues/2929#event-7896601944 at 11/26/2022, 10:43 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1328154344 at 11/26/2022, 06:13 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3730#issuecomment-1328161705 at 11/26/2022, 07:21 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3747 at 11/26/2022, 07:41 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3747#event-7897927808 at 11/27/2022, 09:39 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3704#issuecomment-1328333414 at 11/27/2022, 12:29 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1195113977 at 11/27/2022, 05:58 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3741#pullrequestreview-1196599436 at 11/28/2022, 01:49 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1201445840 at 12/01/2022, 09:57 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3118#issuecomment-1334379811 at 12/01/2022, 12:15 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3694#issuecomment-1334491718 at 12/01/2022, 01:51 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3694#issuecomment-1334543419 at 12/01/2022, 02:48 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3765 at 12/02/2022, 02:18 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3765#issuecomment-1335919447 at 12/02/2022, 02:19 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3765#event-7944266823 at 12/02/2022, 02:19 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3731#issuecomment-1336015108 at 12/02/2022, 06:16 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1336020289 at 12/02/2022, 06:43 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3731#issuecomment-1336202088 at 12/03/2022, 09:22 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1212398142 at 12/09/2022, 06:50 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1212402098 at 12/09/2022, 07:18 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3730#event-8003276908 at 12/09/2022, 07:23 PM PST
- MattPereira closed by PR 3329: https://github.com/hackforla/website/issues/2112#event-8003276962 at 12/09/2022, 07:24 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3767#pullrequestreview-1212436001 at 12/09/2022, 08:21 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3768#issuecomment-1345792133 at 12/11/2022, 06:56 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3768#pullrequestreview-1212851314 at 12/11/2022, 07:13 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3768#event-8009776569 at 12/11/2022, 07:24 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1219911327 at 12/15/2022, 12:36 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3725#pullrequestreview-1220132101 at 12/15/2022, 04:38 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3725#event-8060326656 at 12/16/2022, 02:44 PM PST
- MattPereira closed by PR 3725: https://github.com/hackforla/website/issues/3027#event-8060326721 at 12/16/2022, 02:44 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1369148007 at 01/02/2023, 10:59 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3774#issuecomment-1376381660 at 01/09/2023, 01:56 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3774#pullrequestreview-1241321331 at 01/09/2023, 03:03 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3774#issuecomment-1376475414 at 01/09/2023, 03:17 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3774#event-8186795204 at 01/09/2023, 03:26 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3803 at 01/09/2023, 09:59 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3805 at 01/10/2023, 11:10 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/2728#issuecomment-1377725724 at 01/10/2023, 11:11 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3758#issuecomment-1332402495 at 01/10/2023, 02:57 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3801#issuecomment-1378165402 at 01/10/2023, 06:31 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3767#pullrequestreview-1243220438 at 01/10/2023, 06:45 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3801#pullrequestreview-1244609568 at 01/11/2023, 01:03 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3118#issuecomment-1379503228 at 01/11/2023, 01:25 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3801#pullrequestreview-1244880696 at 01/11/2023, 06:37 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3801#event-8207120218 at 01/11/2023, 06:39 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1383064108 at 01/14/2023, 09:54 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3833#issuecomment-1386045443 at 01/17/2023, 01:01 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3833#pullrequestreview-1252538952 at 01/17/2023, 02:18 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3833#issuecomment-1386299765 at 01/17/2023, 04:48 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3833#pullrequestreview-1252674146 at 01/17/2023, 04:58 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3833#event-8251200847 at 01/17/2023, 05:00 PM PST
- MattPereira closed by PR 3833: https://github.com/hackforla/website/issues/3701#event-8251200944 at 01/17/2023, 05:00 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3835#event-8258310178 at 01/18/2023, 09:53 AM PST
- MattPereira closed by PR 3835: https://github.com/hackforla/website/issues/3398#event-8258310343 at 01/18/2023, 09:53 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3767#pullrequestreview-1260982126 at 01/18/2023, 04:44 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3811#event-8317320800 at 01/19/2023, 04:21 PM PST
- MattPereira closed by PR 3811: https://github.com/hackforla/website/issues/2864#event-8317320951 at 01/19/2023, 04:21 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3858 at 01/22/2023, 01:29 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3858#issuecomment-1399632022 at 01/22/2023, 02:57 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3860 at 01/22/2023, 04:20 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/2571#issuecomment-1399654470 at 01/22/2023, 04:21 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3861 at 01/22/2023, 05:21 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3846#issuecomment-1401088681 at 01/23/2023, 02:23 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3857#issuecomment-1401324523 at 01/23/2023, 06:40 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3857#event-8340130168 at 01/23/2023, 06:40 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3649#pullrequestreview-1267949544 at 01/24/2023, 09:31 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3649#issuecomment-1402328559 at 01/24/2023, 09:32 AM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3649#event-8346687821 at 01/24/2023, 09:34 AM PST
- MattPereira closed by PR 3649: https://github.com/hackforla/website/issues/2626#event-8346688096 at 01/24/2023, 09:34 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3866 at 01/24/2023, 11:21 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3866 at 01/24/2023, 11:21 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3868 at 01/24/2023, 03:04 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3869 at 01/24/2023, 03:35 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3870 at 01/24/2023, 03:45 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3767#pullrequestreview-1268704907 at 01/24/2023, 08:50 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/2572#issuecomment-1401293557 at 01/25/2023, 10:32 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3872 at 01/25/2023, 10:46 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/2572#issuecomment-1404069663 at 01/25/2023, 10:46 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3873 at 01/26/2023, 10:41 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3874 at 01/26/2023, 10:51 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/2354#issuecomment-1401316234 at 01/26/2023, 10:54 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3875 at 01/26/2023, 11:00 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3876 at 01/26/2023, 11:03 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3877 at 01/26/2023, 11:22 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3878 at 01/26/2023, 11:24 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/2354#issuecomment-1405497283 at 01/26/2023, 11:28 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1407554247 at 01/28/2023, 07:57 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3572#issuecomment-1407805861 at 01/29/2023, 05:16 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3572#issuecomment-1407846561 at 01/29/2023, 05:51 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3904 at 01/30/2023, 04:53 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3906 at 01/31/2023, 03:06 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3767#pullrequestreview-1278316720 at 01/31/2023, 06:05 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3767#event-8405489016 at 01/31/2023, 06:05 PM PST
- MattPereira closed by PR 3767: https://github.com/hackforla/website/issues/3547#event-8405489160 at 01/31/2023, 06:05 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3839#issuecomment-1411375413 at 01/31/2023, 06:44 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3905#issuecomment-1411409088 at 01/31/2023, 07:24 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3905#pullrequestreview-1278385929 at 01/31/2023, 07:33 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3888#issuecomment-1411443736 at 01/31/2023, 08:19 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3888#pullrequestreview-1278412529 at 01/31/2023, 08:23 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3883#issuecomment-1411486289 at 01/31/2023, 09:23 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3883#pullrequestreview-1278449929 at 01/31/2023, 09:25 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3883#event-8406528988 at 01/31/2023, 09:26 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3792#issuecomment-1411491689 at 01/31/2023, 09:32 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3792#event-8406555552 at 01/31/2023, 09:32 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3905#event-8406815140 at 01/31/2023, 10:23 PM PST
- MattPereira closed by PR 3905: https://github.com/hackforla/website/issues/3892#event-8406815329 at 01/31/2023, 10:23 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3839#issuecomment-1412554951 at 02/01/2023, 10:52 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3839#pullrequestreview-1279861602 at 02/01/2023, 01:05 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3839#event-8414651432 at 02/01/2023, 01:06 PM PST
- MattPereira closed by PR 3839: https://github.com/hackforla/website/issues/3109#event-8414651640 at 02/01/2023, 01:06 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3921 at 02/01/2023, 01:09 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3921#event-8414728502 at 02/01/2023, 01:17 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3109#issuecomment-1412760899 at 02/01/2023, 01:33 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3109#issuecomment-1412762555 at 02/01/2023, 01:35 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3907#issuecomment-1414205664 at 02/02/2023, 10:49 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3907#pullrequestreview-1281706473 at 02/02/2023, 10:59 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3923#issuecomment-1414232571 at 02/02/2023, 11:10 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3572#issuecomment-1414613821 at 02/02/2023, 05:51 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3895#issuecomment-1415037836 at 02/02/2023, 09:43 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3895#issuecomment-1415041258 at 02/02/2023, 09:45 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3837#issuecomment-1416551657 at 02/03/2023, 04:19 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3888#pullrequestreview-1283955272 at 02/03/2023, 04:30 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3888#event-8437337771 at 02/03/2023, 04:31 PM PST
- MattPereira closed by PR 3888: https://github.com/hackforla/website/issues/3383#event-8437337856 at 02/03/2023, 04:31 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3907#pullrequestreview-1283967967 at 02/03/2023, 04:40 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3907#event-8437364123 at 02/03/2023, 04:40 PM PST
- MattPereira closed by PR 3907: https://github.com/hackforla/website/issues/3818#event-8437364215 at 02/03/2023, 04:40 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3933 at 02/03/2023, 05:20 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3924#issuecomment-1416868407 at 02/04/2023, 02:50 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3924#pullrequestreview-1284123582 at 02/04/2023, 02:56 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3924#event-8439314337 at 02/04/2023, 02:57 PM PST
- MattPereira closed by PR 3924: https://github.com/hackforla/website/issues/3893#event-8439314361 at 02/04/2023, 02:57 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3900#issuecomment-1416265727 at 02/04/2023, 03:05 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3925#issuecomment-1416262079 at 02/04/2023, 03:50 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3925#issuecomment-1416879946 at 02/04/2023, 04:01 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3925#issuecomment-1416879946 at 02/04/2023, 04:02 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3900#issuecomment-1416871780 at 02/04/2023, 04:03 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3903 at 02/04/2023, 06:52 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3931#issuecomment-1416798094 at 02/04/2023, 07:11 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/3936 at 02/04/2023, 07:28 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3903#issuecomment-1416912814 at 02/04/2023, 07:29 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3903#event-8439571511 at 02/04/2023, 07:29 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3903#issuecomment-1416912927 at 02/04/2023, 07:30 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3890#issuecomment-1407747225 at 02/05/2023, 11:12 AM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3931#issuecomment-1416910718 at 02/05/2023, 11:14 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3890#issuecomment-1418252187 at 02/05/2023, 12:04 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3890#issuecomment-1418273561 at 02/05/2023, 01:49 PM PST
- MattPereira opened: https://github.com/hackforla/website/pull/3939 at 02/05/2023, 02:39 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3890#issuecomment-1418288144 at 02/05/2023, 02:57 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3890#issuecomment-1418299816 at 02/05/2023, 03:47 PM PST
- MattPereira PR was merged: https://github.com/hackforla/website/pull/3939#event-8442317708 at 02/05/2023, 04:36 PM PST
- MattPereira issue closed by PR 3939: https://github.com/hackforla/website/issues/3890#event-8442317784 at 02/05/2023, 04:36 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3929#issuecomment-1419568110 at 02/06/2023, 12:54 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3929#issuecomment-1419735155 at 02/06/2023, 01:08 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3901#pullrequestreview-1286214112 at 02/06/2023, 04:26 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3901#event-8453006221 at 02/06/2023, 04:27 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3932#event-8453046992 at 02/06/2023, 04:37 PM PST
- MattPereira closed by PR 3932: https://github.com/hackforla/website/issues/3837#event-8453047120 at 02/06/2023, 04:37 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3352#event-8453440362 at 02/06/2023, 06:15 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1288113515 at 02/07/2023, 02:40 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3887#issuecomment-1421637085 at 02/07/2023, 03:32 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3758#issuecomment-1332402495 at 02/08/2023, 05:09 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3887#issuecomment-1423521294 at 02/08/2023, 06:18 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3936#issuecomment-1424688881 at 02/09/2023, 11:13 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3936#issuecomment-1425064469 at 02/10/2023, 07:36 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3283#issuecomment-1426458346 at 02/10/2023, 03:03 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3283#event-8494438317 at 02/10/2023, 03:03 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3936#issuecomment-1425981528 at 02/10/2023, 03:38 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3334#issuecomment-1427083158 at 02/12/2023, 09:08 AM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3334#issuecomment-1427083158 at 02/12/2023, 09:08 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3961#issuecomment-1427098854 at 02/12/2023, 10:22 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1427107847 at 02/12/2023, 11:02 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3812#issuecomment-1427120341 at 02/12/2023, 12:04 PM PST
- MattPereira closed: https://github.com/hackforla/website/pull/3812#event-8498928690 at 02/12/2023, 12:04 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1294792656 at 02/12/2023, 02:15 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1294794444 at 02/12/2023, 02:26 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3965#pullrequestreview-1297861959 at 02/14/2023, 07:38 AM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3965#event-8516921123 at 02/14/2023, 07:41 AM PST
- MattPereira closed by PR 3965: https://github.com/hackforla/website/issues/3851#event-8516921860 at 02/14/2023, 07:41 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1298220600 at 02/14/2023, 10:49 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1298567546 at 02/14/2023, 03:19 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1298624861 at 02/14/2023, 04:19 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3887#event-8520892692 at 02/14/2023, 04:20 PM PST
- MattPereira closed by PR 3887: https://github.com/hackforla/website/issues/3256#event-8520892793 at 02/14/2023, 04:20 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3978#issuecomment-1431653701 at 02/15/2023, 08:36 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3977#issuecomment-1431687209 at 02/15/2023, 08:59 AM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3977#pullrequestreview-1299932946 at 02/15/2023, 09:02 AM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3977#event-8528297780 at 02/15/2023, 09:03 AM PST
- MattPereira closed by PR 3977: https://github.com/hackforla/website/issues/3694#event-8528298147 at 02/15/2023, 09:03 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/3978#issuecomment-1432089101 at 02/15/2023, 01:53 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/3978#pullrequestreview-1302603317 at 02/16/2023, 06:06 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3978#event-8542861532 at 02/16/2023, 06:07 PM PST
- MattPereira closed by PR 3978: https://github.com/hackforla/website/issues/3110#event-8542861641 at 02/16/2023, 06:07 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4003#pullrequestreview-1304331571 at 02/17/2023, 04:19 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4003#pullrequestreview-1304336237 at 02/17/2023, 04:23 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3989#event-8551942121 at 02/17/2023, 04:28 PM PST
- MattPereira closed by PR 3989: https://github.com/hackforla/website/issues/3873#event-8551942183 at 02/17/2023, 04:28 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3993#event-8552105263 at 02/17/2023, 05:54 PM PST
- MattPereira closed by PR 3993: https://github.com/hackforla/website/issues/3944#event-8552105307 at 02/17/2023, 05:54 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/3995#event-8552395381 at 02/17/2023, 09:24 PM PST
- MattPereira closed by PR 3995: https://github.com/hackforla/website/issues/3822#event-8552395409 at 02/17/2023, 09:24 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4004#event-8552398762 at 02/17/2023, 09:27 PM PST
- MattPereira closed by PR 4004: https://github.com/hackforla/website/issues/3955#event-8552398803 at 02/17/2023, 09:27 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4005#pullrequestreview-1304481034 at 02/17/2023, 09:37 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4005#event-8552414771 at 02/17/2023, 09:40 PM PST
- MattPereira closed by PR 4005: https://github.com/hackforla/website/issues/3936#event-8552414795 at 02/17/2023, 09:40 PM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3361#event-8555767033 at 02/19/2023, 10:42 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4023 at 02/22/2023, 07:37 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4026 at 02/22/2023, 07:42 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4031 at 02/23/2023, 08:27 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4027#issuecomment-1444613308 at 02/24/2023, 02:49 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4027#pullrequestreview-1314355750 at 02/24/2023, 02:58 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4027#event-8606329762 at 02/24/2023, 02:59 PM PST
- MattPereira closed by PR 4027: https://github.com/hackforla/website/issues/3802#event-8606329940 at 02/24/2023, 02:59 PM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3866#issuecomment-1444723224 at 02/24/2023, 03:41 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4036 at 02/24/2023, 04:08 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4037 at 02/24/2023, 04:10 PM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/3879#issuecomment-1442853097 at 02/24/2023, 11:23 PM PST
- MattPereira unassigned: https://github.com/hackforla/website/issues/3879#issuecomment-1442853097 at 02/25/2023, 09:04 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/3879#issuecomment-1445160795 at 02/25/2023, 09:08 AM PST
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3879#event-8608196447 at 02/25/2023, 09:08 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4040 at 02/25/2023, 09:32 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4041 at 02/25/2023, 09:40 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4042 at 02/25/2023, 10:21 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4043 at 02/25/2023, 10:30 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4044 at 02/25/2023, 10:34 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4049 at 02/25/2023, 02:33 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4034#pullrequestreview-1314723281 at 02/26/2023, 12:15 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4034#event-8610114438 at 02/26/2023, 12:16 PM PST
- MattPereira closed by PR 4034: https://github.com/hackforla/website/issues/3931#event-8610114477 at 02/26/2023, 12:16 PM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4054 at 02/28/2023, 08:29 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/4054 at 02/28/2023, 08:31 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4055 at 02/28/2023, 09:48 AM PST
- MattPereira commented on: https://github.com/hackforla/website/issues/4055#issuecomment-1448641396 at 02/28/2023, 10:08 AM PST
- MattPereira assigned: https://github.com/hackforla/website/issues/4055#issuecomment-1448641396 at 02/28/2023, 10:08 AM PST
- MattPereira opened: https://github.com/hackforla/website/pull/4057 at 02/28/2023, 10:21 AM PST
- MattPereira PR was merged: https://github.com/hackforla/website/pull/4057#event-8630188194 at 02/28/2023, 10:37 AM PST
- MattPereira issue closed by PR 4057: https://github.com/hackforla/website/issues/4055#event-8630188473 at 02/28/2023, 10:37 AM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4091#event-8664206956 at 03/03/2023, 03:00 PM PST
- MattPereira closed by PR 4091: https://github.com/hackforla/website/issues/3963#event-8664207250 at 03/03/2023, 03:00 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4092#event-8664214031 at 03/03/2023, 03:02 PM PST
- MattPereira closed by PR 4092: https://github.com/hackforla/website/issues/4044#event-8664214172 at 03/03/2023, 03:02 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4096#issuecomment-1454247401 at 03/03/2023, 03:10 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4096#issuecomment-1454248854 at 03/03/2023, 03:12 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4096#pullrequestreview-1324738672 at 03/03/2023, 03:16 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4096#event-8664309450 at 03/03/2023, 03:16 PM PST
- MattPereira closed by PR 1: https://github.com/hackforla/website/issues/4045#event-8664309573 at 03/03/2023, 03:16 PM PST
- MattPereira opened: https://github.com/hackforla/website/pull/4105 at 03/03/2023, 04:47 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4078#issuecomment-1454339986 at 03/03/2023, 06:25 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4093#event-8666406999 at 03/04/2023, 08:32 AM PST
- MattPereira closed by PR 4093: https://github.com/hackforla/website/issues/4037#event-8666407035 at 03/04/2023, 08:32 AM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4110#issuecomment-1454909603 at 03/04/2023, 02:39 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4078#pullrequestreview-1325143610 at 03/04/2023, 05:42 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4100#issuecomment-1455198898 at 03/05/2023, 12:38 PM PST
- MattPereira submitted review: https://github.com/hackforla/website/pull/4100#pullrequestreview-1325276337 at 03/05/2023, 12:45 PM PST
- MattPereira merged: https://github.com/hackforla/website/pull/4100#event-8668654950 at 03/05/2023, 12:45 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4105#issuecomment-1456584292 at 03/06/2023, 09:27 AM PST
- MattPereira opened: https://github.com/hackforla/website/issues/4145 at 03/09/2023, 11:10 PM PST
- MattPereira commented on: https://github.com/hackforla/website/pull/4105#issuecomment-1468887288 at 03/14/2023, 02:47 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/4110#pullrequestreview-1347315313 at 03/18/2023, 04:45 PM PDT
- MattPereira merged: https://github.com/hackforla/website/pull/4110#event-8786486927 at 03/18/2023, 04:47 PM PDT
- MattPereira closed by PR 4110: https://github.com/hackforla/website/issues/3341#event-8786487020 at 03/18/2023, 04:47 PM PDT
- MattPereira opened: https://github.com/hackforla/website/issues/4219 at 03/20/2023, 09:08 AM PDT
- MattPereira PR was merged: https://github.com/hackforla/website/pull/4105#event-8801094135 at 03/20/2023, 08:53 PM PDT
- MattPereira issue closed by PR 4105: https://github.com/hackforla/website/issues/4054#event-8801094205 at 03/20/2023, 08:53 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/4105#issuecomment-1477727345 at 03/21/2023, 05:08 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4040#issuecomment-1478615764 at 03/21/2023, 02:37 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4040#issuecomment-1478617255 at 03/21/2023, 02:39 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/3758#issuecomment-1480000963 at 03/22/2023, 10:42 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/3758#issuecomment-1480038829 at 03/22/2023, 11:10 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/4361#pullrequestreview-1368017916 at 04/01/2023, 09:26 PM PDT
- MattPereira merged: https://github.com/hackforla/website/pull/4361#event-8905724365 at 04/01/2023, 09:28 PM PDT
- MattPereira closed by PR 4361: https://github.com/hackforla/website/issues/2220#event-8905724379 at 04/01/2023, 09:28 PM PDT
- MattPereira opened: https://github.com/hackforla/website/issues/4370 at 04/02/2023, 10:17 AM PDT
- MattPereira unassigned: https://github.com/hackforla/website/issues/2354#issuecomment-1405497283 at 04/02/2023, 10:34 AM PDT
- MattPereira unassigned: https://github.com/hackforla/website/issues/2572#issuecomment-1404069663 at 04/02/2023, 10:35 AM PDT
- MattPereira opened: https://github.com/hackforla/website/issues/4392 at 04/03/2023, 03:55 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4392#issuecomment-1496212245 at 04/04/2023, 08:48 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1498097880 at 04/05/2023, 01:25 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1498141687 at 04/05/2023, 01:54 PM PDT
- MattPereira closed as completed: https://github.com/hackforla/website/issues/3858#event-8950278639 at 04/06/2023, 03:50 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4392#issuecomment-1506010357 at 04/12/2023, 02:54 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/4602#issuecomment-1536448081 at 05/05/2023, 08:46 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/4602#pullrequestreview-1415049097 at 05/05/2023, 08:53 AM PDT
- MattPereira unassigned: https://github.com/hackforla/website/issues/3866#issuecomment-1547100517 at 05/14/2023, 07:15 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4868#issuecomment-1605688180 at 06/24/2023, 12:00 PM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4868#issuecomment-1605730685 at 06/24/2023, 02:01 PM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/4899#pullrequestreview-1505489961 at 06/29/2023, 07:57 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/5037#pullrequestreview-1539706873 at 07/20/2023, 09:56 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/pull/5037#issuecomment-1650085006 at 07/25/2023, 08:42 AM PDT
- MattPereira submitted review: https://github.com/hackforla/website/pull/5037#pullrequestreview-1548114288 at 07/26/2023, 09:05 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/5086#issuecomment-1656819128 at 07/29/2023, 11:26 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/4906#issuecomment-1656825623 at 07/29/2023, 11:38 AM PDT
- MattPereira commented on: https://github.com/hackforla/website/issues/5219#issuecomment-1690504120 at 08/23/2023, 12:13 PM PDT
